addons/im_livechat/static/src/messaging/entity/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -0,0 +1,35 @@
+odoo.define('im_livechat.messaging.entity.MessagingNotificationHandler', function (require) {
+'use strict';
+
+const { registerInstancePatchEntity } = require('mail.messaging.entityCore');
+
+registerInstancePatchEntity('MessagingNotificationHandler', 'im_livechat.messaging.entity.MessagingNotificationHandler', {
+
+    //----------------------------------------------------------------------
+    // Private
+    //----------------------------------------------------------------------
+
+    /**
+     * @override
+     * @param {Object} param0
+     * @param {integer} param0.channelId
+     * @param {integer} param0.partner_id
+     */
+    _handleNotificationChannelTypingStatus(data) {
+        const { channelId, partner_id } = data;
+        const channel = this.env.entities.Thread.insert({ id: channelId, model: 'mail.channel' });
```

Comment:
convertData probably

Reply:
`convertData` is a parser of `channel_info`, and data do not come from this formatter so there's no reason to use `convertData`. Also 2 data bits do not motivate having a converter, especially when that's very specific to this LOC.

Reply:
ok